lib/tasks/scripts/purge_users_data.rb

```diff
@@ -0,0 +1,16 @@
+module PurgeUsersData
+  def self.perform
+    return if ENV['SIMPLE_SERVER_ENV'] == 'production'
+
+    models = [BloodPressure, BloodSugar, Appointment, CallLog, Communication, PrescriptionDrug,
```

Comment:
I'm guessing the order of models here is important to avoid FK violations. If so, can you leave a comment stating as such?

Also, for formatting long lists of Arrays like this, I would recommend doing one model per line:

```
models = [
  BloodPressure,
  BloodSuguar,
  Appointment,
... etc
]
```

It will make diffs and changing this list much easier over time.

Reply:
[Addressed here.](https://github.com/simpledotorg/simple-server/pull/786/commits/8c30c3878505e0a30003dfc87ab64c870b2bc448)